### PR TITLE
every 24 hours obsolete results from DB get deleted so there is only …

### PR DESCRIPTION
**Pull request by Patross**
…data from past 7 days displayed

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,6 +24,40 @@ client.once('ready', () => {
...
+				Math.floor(d1.getTime()/ 1000)
+				var date = new Date( d1.getUTCFullYear(), d1.getUTCMonth(), d1.getUTCDate(), d1.getUTCHours(), d1.getUTCMinutes(), d1.getUTCSeconds() );
+				const oneDay = 24 * 60 * 60 * 1000; // hours*minutes*seconds*milliseconds
+	
+	
+				
+				results.forEach(element => {
+					const diffDays = Math.round(Math.abs((element.date - date) / oneDay));
+					let D = element.date;
+					// D.setDate(D.getDate());
```
Comment (AndyTheNerd) [inline index.js:44]: Do we need this comment?

File: index.js [hunk 2]
```diff
@@ -24,6 +24,40 @@ client.once('ready', () => {
...
+						connection.query(`DELETE FROM ${process.env.mysql_command_stats_table} WHERE date = "${new Date(Date.parse(D)).toISOString().slice(0, 10).replace('T', ' ')}"`,function(err,result){
+							if (err) {
+								console.log(err);
+								console.log(`Something went wrong. If the issue persists contact the developers. \`!support\``);
+							}else{
+
+							}
+						})
+					} 
+						// console.log(element.date);
```
Comment (AndyTheNerd) [inline index.js:55]: Or this one?

**Comment by Patross** [comment]
Nope

**Comment by Patross** [comment]
removed comments

**Pull request merged**